OpenIGTLinkRemote/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(MODULE_NAME "OpenIGTLinkRemote" )
 set(MODULE_TITLE "OpenIGTLink Remote" )
 
 string(TOUPPER ${MODULE_NAME} MODULE_NAME_UPPER)
-
+set(OpenIGTLinkIF_DIR ${SlicerOpenIGTLink_DIR})
```

Comment:
This is odd. It should not be necessary. Package name should be SlicerOpenIGTLink, too.